python/tvm/relax/op/nn/nn.py

```diff
@@ -341,6 +341,34 @@ def softmax(data: Expr, axis: int = -1) -> Expr:
     return _ffi_api.softmax(data, axis)  # type: ignore
 
 
+def log_softmax(data: Expr, axis: int = -1) -> Expr:
+    r"""Computes log softmax.
+
+    .. math::
+
+        \text{log_softmax}(x)_i = \log \frac{exp(x_i)}{\sum_j exp(x_j)}
```

Comment:
We could write the math doc according to torch: <https://pytorch.org/docs/stable/generated/torch.nn.LogSoftmax.html>

```suggestion
        \text{log\_softmax}(x_i) = \log\left( \frac{\exp(x_i)}{\sum_j \exp(x_j)}\right)
```

It seems that the math doc of `softmax` also have some problem: it is lack of backslash, and should use `\text{softmax}(x_i)`. Maybe we could fix that as well?

Reply:
If so, there are many math doc needed to be fixed.